modules/activiti-engine/src/test/resources/org/activiti/engine/test/history/HistoricTaskInstanceUpdateTest.testHistoricTaskInstanceExecutionIdUpdateOnConcurrentExecution.bpmn20.xml

```diff
@@ -0,0 +1,123 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The connection destination of the sequence flow appears to be wrong.

Reply:
For example, you can check your bpmn by uploading to activiti-app.
The following image is your bpmn(HistoricTaskInstanceUpdateTest.testHistoricTaskInstanceExecutionIdUpdateOnConcurrentExecution.bpmn20.xml).
![2017-12-18 22 42 55](https://user-images.githubusercontent.com/16251291/34108847-08590180-e445-11e7-8399-9908d04a37ab.png)

Reply:
Yeah, i have removed the bpmndi XML tags thinking that it is a test case. Do you want me add them ?

Reply:
I see. Sequence flow connection seems to have no problem. I'm sorry.

But, If possible, can not you make it a more descriptive ID? It is hard to understand if it is an automatically generated ID.

* start -> sid-6A656DD6-E0BE-4D05-BE5C-0039AD869B07
* sid-6A656DD6-E0BE-4D05-BE5C-0039AD869B07 -> parallelgate
  * parallelgate -> UserA
    * UserA -> sid-4D68D7EC-04A7-47BA-BA66-3AB1216E0497
      * sid-4D68D7EC-04A7-47BA-BA66-3AB1216E0497 -> UserB
        * UserB -> sid-6A656DD6-E0BE-4D05-BE5C-0039AD869B07
      * sid-4D68D7EC-04A7-47BA-BA66-3AB1216E0497 -> sid-93AA0B2A-8454-457A-BA31-A19DDF32069D
        * sid-93AA0B2A-8454-457A-BA31-A19DDF32069D -> sid-6A656DD6-E0BE-4D05-BE5C-0039AD869B07
  * parallelgate -> UserC
    * UserC -> End2